.gitlab-ci.yml

```diff
@@ -65,6 +69,22 @@ Test Edge 84:
       - RUNNER:
           - openstack/rhel-8-x86_64
 
+Test Load Test:
+  stage: test
+  extends: .terraform
+  rules:
+    - if: '$CI_PIPELINE_SOURCE != "schedule"'
+  script:
+    - schutzbot/ci_details.sh
+    - schutzbot/build.sh
+    - schutzbot/deploy.sh
```

Comment:
As discussed we can probably drop some of these.

Reply:
I'm on it!